core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -726,6 +726,7 @@ public enum Property {
       "The number of threads on each tablet server available to retrieve"
           + " summary data, that is not currently in cache, from RFiles.",
       "2.0.0"),
+  @Deprecated(since = "3.1", forRemoval = true)
```

Comment:
The description of this property could benefit from an explanation for its deprecation/what replaces it, so users don't just see that it's deprecated in the generated docs and wonder why.

Reply:
Is this a general comment, or just for this property? If general, then I should probably update the other properties that are already marked as deprecated.

Reply:
I was thinking narrowly about this one property, but if there are others, then that should be done, too, but this is the only one in scope of this PR.

Reply:
Updated description for all deprecated properties in 7d663d0